### ISSUE TITLE
Property `rendering/quality/driver/driver_name` not found on Linux

### ISSUE THREAD

**Issue by paulloz** [godotengine/godot#17190]
**Godot version:** `master` / 4ccc8e5

**OS/device including version:** *Linux Mint 18.3*

**Issue description:**
I'm getting this error in every project I load or run in the editor. It does not seem to break anything though.
```
WARNING: not found: rendering/quality/driver/driver_name
ERROR: set_custom_property_info: Condition ' !props.has(p_prop) ' is true.
   At: core/project_settings.cpp:847.
```
Probably related to eac4c98.

**Comment by akien-mga** [comment]
Should be fixed by #17158.

**Comment by paulloz** [comment]
Indeed :tada:
